cmake/FindWPE.cmake

```diff
@@ -49,20 +50,44 @@ if(PC_WPE_FOUND)
         NAMES ${PC_WPE_LIBRARIES}
         HINTS ${PC_WPE_LIBDIR} ${PC_WPE_LIBRARY_DIRS}
     )
-
 else()
     message(FATAL_ERROR "libwpe not found!")
 endif()
 
+if (WPE_INCLUDE_DIR AND NOT WPE_VERSION)
+    if (EXISTS "${WPE_INCLUDE_DIR}/wpe/version.h")
+        file(READ "${WPE_INCLUDE_DIR}/wpe/version.h" WPE_VERSION_CONTENT)
+
+        string(REGEX MATCH "#define +WPE_MAJOR_VERSION +\\(([0-9]+)\\)" _dummy "${WPE_VERSION_CONTENT}")
```

Comment:
I'm not a cmake person but this seems pretty fragile, do we not have a pkgconfig file to query?

Reply:
Yes, the version will be picked from `wpe-1.0.pc` file, and this bit only used if the `.pc` file did not include a version (highly unlikely) or not present. The latter could happen, apparently that's a thing in Windows; and also the reason why we have this in the `FindWPE.cmake` file in the WebKit tree (from which I picked it).

On second thought, we could actually remove this part from the patch, as it is safe to assume that `WPEBackend-rdk` will always be built on Linux.

Reply:
Done, updated!